packages/rpc-core/src/rpc-methods/getClusterNodes.ts

```diff
@@ -0,0 +1,37 @@
+import { Base58EncodedAddress } from '@solana/keys';
+
+type GetClusterNodesNode = Readonly<{
+    /** Node public key, as base-58 encoded string */
+    pubkey: Base58EncodedAddress;
+    /** Gossip network address for the node */
+    gossip: string | null;
+    /** TPU network address for the node */
+    tpu: string | null;
+    /**
+     * JSON RPC network address for the node,
+     * or `null` if the JSON RPC service is not enabled
+     */
+    rpc: string | null;
+    /**
+     * The software version of the node,
+     * or `null` if the version information is not available
+     */
+    version: string | null;
+    /** The unique identifier of the node's feature set */
+    featureSet: number | null;
+    /** The shred version the node has been configured to use */
+    shredVersion: number | null;
```

Comment:
Wondering why lint didn't scream at you to alpha-order these.

Alpha order these?